File: Web2Backend/Web2Backend/Model/AirCompany.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Web2Backend.Model
{
    public class AirCompany
    {
        [Key]
        public int AirCompanyId { get; set; }
        public String Name { get; set; }

        public String Address { get; set; }
        public String PromoDescription { get; set; }

        public double MeanGrade { get; set; }

        public ICollection<FlightAdmin> FlightAdmins { get; set; }
        public ICollection<SoldTicket> SoldTickets { get; set; }
        public ICollection<Flight> Flights { get; set; }
        public ICollection<Luggage> Luggage { get; set; }
        public AirCompany()
        {
            this.FlightAdmins = new HashSet<FlightAdmin>();
            this.SoldTickets = new HashSet<SoldTicket>();
            this.Flights = new HashSet<Flight>();
            this.Luggage = new HashSet<Luggage>();

        }
        //arrays: flightGrade
    }
}

File: Web2Proj/src/app/entities/airCompany.ts
import { Flight } from 'src/app/shared/flight-service-and-model/flight';
import { FlightAdmin } from './flightAdmin';
import { SoldTicket } from './sold-ticket';
import { Luggage } from './luggage';

export class AirCompany{
    airCompanyId: number;
    name: string;
    address: string;
    promoDescription: string;
    meanGrade: number;    
}
